### DISCUSSION TITLE
Excel

### DISCUSSION THREAD

**Discussion by Ashu8009** [sqlitebrowser/sqlitebrowser#3502] [General]
![Sample](https://github.com/sqlitebrowser/sqlitebrowser/assets/131003079/fe5cbada-7bc5-497c-9010-da8fcf09ff6d)
hello members,
i created one desktop inventory for the records where 2 collum were added like start warranty and end warranty please guide me i need to auto-update this collum, like system is under warranty its showing green please guide me

**Comment by justinclift** [comment]
Ahhh.  You'll probably need to use some external program to modify the data.

It *sounds* like there is some kind of um... events happening (eg change of date), which can mean things like the warranty status changes on that date.

If that's the case, then it's the sort of thing you'd do using a programming language for.

Are you any good at programming, or at least willing to learn? :smile:

**Reply by chrisjlocke** [reply]
I did look at conditional formatting, as theoretically you could highlight a cell red if its in the past, and while it sort of worked using ```warrantyDate < '2023-11-01'``` I couldn't get it working with ```warrantyDate < date()```

**Reply by justinclift** [reply]
It later on occurred to me that this discussion is titled "Excel", so may not really be about DB4S stuff.  Could just be something accidentally created in the wrong repo. :wink:

**Reply by chrisjlocke** [reply]
I think its more a shortened version of 'I've an Excel spreadsheet here that does X .. how can DB4S do the same thing....'
